### DISCUSSION TITLE
Why are some of these services spooky ? 👻

### DISCUSSION THREAD

**Discussion by fredrikwestman92** [frenck/spook#450] [Documentation]
Just found this integration. Looks really awesome!

Reading through the documentation got me a bit curious tho, as to why some of these features didn't make the official cut so to speak.

Specifically the input_number and the input_select services 🤔? Seems harmless enough and very useful.

**Comment by frenck** [comment]
🤷 it all depends.

As per documentation of Spook, some things might eventually end up in Core itself, others maybe not. 🤷 


../Frenck

**Comment by fredrikwestman92** [comment]
Okay sorry I must have missed that part 🙂
